File: app/src/main/java/com/cookplan/upload_image/UploadImageView.java
package com.cookplan.upload_image;

/**
 * Created by DariaEfimova on 13.06.17.
 */

public interface UploadImageView {
    void setError(String error);

    void setImageSaved(String url);

    void setImageRemoved(String imageId);
}

File: app/src/main/java/com/cookplan/upload_image/UploadImageView.kt
package com.cookplan.upload_image

/**
 * Created by DariaEfimova on 13.06.17.
 */

interface UploadImageView {

    fun setError(error: String)

    fun setImageSaved(imageId: String)

    fun setImageRemoved(imageId: String)
}